### PR TITLE
feat: configure Tailwind CSS

### LINKED ISSUE #7: Configure Tailwind CSS
## Description
This task focuses on revising our Tailwind CSS configuration to align with the latest design specifications provided by our design team in Figma.

## Criteria

- [ ] Add fonts
- [ ] Add colors
- [ ] Add vector icons (if needed)

## Notes

- Here is the link to [our Figma](https://www.figma.com/file/5tNJDDfqC0HnHbWygzEXiL/TEST?type=design&node-id=0-1&mode=design&t=wA08h7KevTTFanaI-0)

### PR DESCRIPTION
**Pull request by HrRn**
## Overview

I configured Tailwind CSS. There are uncertain colors, so it may be necessary to confirm with the designer.

## Changes

- [x] Add fonts
- [x] Add colors
- [ ] Add vector icons (if needed)

## Assignee Checklist:

- [x] The base branch is correct (no accidental merges)
- [x] The branch name follows our branch naming rules
- [x] The PR title follows our PR title rules
- [x] My code follows our coding style
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings or errors in the console

## Reviewer Checklist:

- [x] Can use classes correctly

### CODE REVIEW

File: tailwind.config.ts [hunk 1]
```diff
@@ -1,20 +1,34 @@
...
+        destructive: {
+          DEFAULT: 'rgb(var(--color-destructive))',
+        },
+        accent: {
+          DEFAULT: 'rgb(var(--color-accent))',
+        },
+      },
+
+      fontFamily: {
+        sans: ['Roboto', 'sans-serif'],
```
Comment (akiomatic) [inline tailwind.config.ts]: Doesn't this have to be "roboto", not "sans"?
Also, the sub-font might be "latin", referring from `layout.tsx`.

**Review by akiomatic — CHANGES_REQUESTED** [review]
Thank you for the configuration!
I'd like you to check the comments that I left!

**Comment by akiomatic** [comment]
@HrRn 
Have you merged `develop` into this branch?

**Comment by akiomatic** [comment]
@HrRn 
Please let me know when I'm ready to review this PR.

**Review by akiomatic — APPROVED** [review]
LGTM!

File: src/components/ui/button.tsx [hunk 2]
```diff
@@ -1,56 +1,56 @@
-import * as React from "react"
-import { Slot } from "@radix-ui/react-slot"
-import { cva, type VariantProps } from "class-variance-authority"
+import * as React from 'react';
```
Comment (Akikaze1119) [inline src/components/ui/button.tsx]: Thank you for configuring!

According to the 'Style Guide', it advises against using wildcard imports. Therefore, I believe it's better to modify and adhere to this guideline.

// Bad
`import * as React from 'react'`

// Good
`import { ReactNode } from 'react';`

Comment (HrRn) [reply src/components/ui/button.tsx]: Thank you for your review!
I modified it!

**Review by Akikaze1119 — APPROVED** [review]
Thank you! LGTM ;)

**Pull request merged**